courses/fundamentals_of_ada/020_declarations.rst

```diff
@@ -619,33 +615,27 @@ Name Hiding
 Overcoming Hiding
 -------------------
 
-.. container:: columns
+* Add a **prefix**
 
- .. container:: column
-  
-    * Based on providing additional name qualifiers
+   - Needs named scope
 
-       - Works for any named scope
+* Homographs are a *code smell*
```

Comment:
Never heard that term before (but it seems to be somewhat common, so you can leave it :)